### PR TITLE
GCSMultiBatchSink description changed

### PR DESCRIPTION
**Pull request by vjosadika**
Jira ticket https://issues.cask.co/browse/PLUGIN-232

### CODE REVIEW

**Comment by vjosadika** [comment]
@googlebot I signed it!

**Review by CuriousVini — APPROVED** [review]

File: src/main/java/io/cdap/plugin/gcp/gcs/sink/GCSMultiBatchSink.java [hunk 1]
```diff
@@ -52,7 +52,7 @@
  */
 @Plugin(type = BatchSink.PLUGIN_TYPE)
 @Name("GCSMultiFiles")
-@Description("Writes records to one or more avro files in a directory on Google Cloud Storage.")
+@Description("Writes records to one or more Avro, ORC, Parquet or Delimited format files in a directory on Google Cloud Storage.")
```
Comment (CuriousVini) [inline src/main/java/io/cdap/plugin/gcp/gcs/sink/GCSMultiBatchSink.java]: Does this plugin compile with `mvn clean package -DskipTests`?

Comment (vjosadika) [reply src/main/java/io/cdap/plugin/gcp/gcs/sink/GCSMultiBatchSink.java]: Fixed

**Review by CuriousVini — APPROVED** [review]
lgtm 
Please squash the commits.

**Pull request merged**